modules/s3db/deploy.py

```diff
@@ -1193,9 +1212,26 @@ def deploy_rheader(r, tabs=[], profile=False):
                                TH("%s: " % table[f].label, **attr)
                 value = lambda f, table=table, record=record, **attr: \
                                TD(table[f].represent(record[f]), **attr)
+
+                def event_type_link_label():
+                    db = current.db
+                    return TH(db
+                              .deploy_mission_event_type["event_type_id"]
+                              .label)
+
+                def event_type_link_value():
+                    db = current.db
+                    row = db(
+                        db.deploy_mission_event_type.mission_id
+                        == record.id).select().first()
```

Comment:
Select only the things you need(i.e, only event_type_id) and you could put limitby = (0, 1). Saves an extra function call.